rel/pattern_set.go

```diff
@@ -73,15 +73,20 @@ func (p SetPattern) Bind(local Scope, value Value) (Scope, error) {
 			if _, is := t.Expr.(IdentExpr); !is {
 				return EmptyScope, fmt.Errorf("item type %s is not supported yet", t)
 			}
-		case IdentPattern:
-			v, has := local.Get(t.ident)
-			if !has {
-				return EmptyScope, fmt.Errorf("%q not in scope", t.ident)
-			}
-			if !set.Has(v.(Value)) {
-				return EmptyScope, fmt.Errorf("item %s is not included in set %s", v, value)
+		case ExprsPattern:
+			// Support cases:
+			// AssertCodesEvalToSameValue(t, `{5, 6}`, `let x = 1; let y = 42; let {(x), (y), ...t} = {1, 42, 5, 6}; t`)
+			// AssertCodeErrors(t, `let x = 1; let y = 42; let {(x), (y)} = {1, 4}; 2`, "")
```

Comment:
```suggestion
			// AssertCodeErrors(t, "", `let x = 1; let y = 42; let {(x), (y)} = {1, 4}; 2`)
```
Nit: expected before actual

Reply:
Will fix in https://github.com/arr-ai/arrai/pull/475